src/Plivo/Resources/Conference/ConferenceInterface.php

```diff
@@ -61,7 +61,9 @@ public function get($conferenceName)
             $optionalArgs
         );
 
-
+        if($response->getThrownException() != null) {
+            throw $response->getThrownException();
+        }
```

Comment:
This would be shorter and more idiomatic:
```suggestion
        if(!$response->ok()) {
            throw $response->getThrownException();
        }
```